lib/ui/tabs/society_page.dart

```diff
@@ -76,6 +76,10 @@ class SocietyDetailPageState extends State<SocietyDetailPage> {
                     },
                     child: CachedNetworkImage(
                       imageUrl: widget.snapshot['image'],
+                      errorWidget: (context, url, error) => Icon(
+                        Icons.broken_image,
+                        color: Colors.grey,
```

Comment:
This is somewhere you can add the `const` keyword. (`Icon`)
Making a widget which is used often `const`, improves performance.

Reply:
Done